contrib/startback-jobs/lib/startback/jobs.fio

```diff
@@ -17,6 +17,8 @@ Job.Full = {
   opContext       : DumpableHash
   opResult        : .
   isReady         : Boolean
+  hasFailed       : Boolean
+  opError         : String|Nil
```

Comment:
We probably need to make sure that this is backward compatible.
If existing jobs (e.g. in Redis) are dressed, that might fail. We must introduce versioning and information contracts to upgrade.